ros_buildfarm/templates/snippet/publisher_disable-failed-job.xml.em

```diff
@@ -1,5 +1,5 @@
     <disableFailedJob.disableFailedJob.DisableFailedJob plugin="disable-failed-job@@1.15">
-    <whenDisable>Only Failure</whenDisable>
-    <failureTimes>@(failure_times)</failureTimes>
-    <optionalBrockChecked>true</optionalBrockChecked>
+      <whenDisable>Only Failure</whenDisable>
+      <failureTimes>@(failure_times)</failureTimes>
+      <optionalBrockChecked>true</optionalBrockChecked>
     </disableFailedJob.disableFailedJob.DisableFailedJob>
```

Comment:
If I had to guess the old indentation was the result of a buggy
serializer which has been fixed by the xtream changes now that we've
updated to Jenkins 2.277.3.